webapp/src/containers/LinkModal/index.tsx

```diff
@@ -71,9 +72,9 @@ const LinkModal = () => {
 
         // Make POST api request
         linkTask(projectDetails);
-    }, [errorState]);
+    };
 
-    // Make POST API request to link a project
+    // Make POST api request to link a project
```

Comment:
Why this change?

Reply:
this was a comment by some other reviewer on any of the PR, and I thought it's good to use caps for abbreviations

Reply:
I think API is better.